lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journeys/strategic-app-triage.yaml

```diff
@@ -56,6 +56,7 @@ nestedJourneyStates:
     response:
       type: process
       lambda: call-dcmaw-async-cri
+      context: dad
```

Comment:
Ohh, we do have a `lambdaInput` thing we use in other states - is that something we should/could use here too instead?

Reply:
Ah yes, I can use that. Context felt more appropriate given the name but lambdaInput seems to be used for all sorts. If you think lambdaInput is more appropriate I can go about removing the context parameterisation